### ISSUE TITLE
config_archive.xml syntax error

### ISSUE THREAD

**Issue by jedwards4b** [ESMCI/cime#2194]
Line 123 has ```  <hist_file_extension>\..+$_obs_seq.*</hist_file_extension>```
I think it should be ```  <hist_file_extension>\..+\_obs_seq.*</hist_file_extension>```

**Comment by jedwards4b** [comment]
```Please reassign this CIME issue to Kevin Raeder. It is related to DART file archival and we have
no way to test DART simulations in CIME that I know of.```
You can do that yourself on the github page.

**Comment by kdraeder** [comment]
@jedwards4b @bertinia 
The best version at this point is 
<hist_file_extension>\..+obs_seq.*</hist_file_extension>

In the next few weeks I'll want to upgrade this DART section
of config_archive.xml to handle our new file naming convention.

**Comment by jedwards4b** [comment]
@kdraeder Do you want to put this fix in now or wait for your DART upgrade?

**Comment by kdraeder** [comment]
I'm not quite ready yet.

**Comment by kdraeder** [comment]
The DART obs_seq files have CESM-format names in the
recently finished upgrades to st_archive and related files,
so they don't require special treatment in config_archive.xml.
This issue is resolved by #2338, and #2415.

**Comment by kdraeder** [comment]
@jedwards4b 
I believe that this issue can be closed.
